README.md

```diff
@@ -129,6 +130,45 @@ Run SortingHat frontend Vue app:
 $ yarn serve
 ```
 
+
+## SortingHat service
+
+Starting on version 0.8, SortingHat is released with a server app. The server has two
```

Comment:
This should  be _starting **at** version 0.8_

Reply:
Fixed.